src/components/review/Review.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+    import { _ } from "svelte-i18n";
+    
+    export let name: string;
+    export let body: string;
+    export let created_at: string;
+    export let clickable: boolean = false;
+    export let short_body_char_limit: number = 150;
+    export let link: string = "/review";
+
+    let time_ago = (created_at: string): string => {
+        let date = new Date(created_at);
+        let todayDate = new Date();
+        let timeDiff = Math.abs(todayDate.getTime() - date.getTime());
+        let diffDays = Math.floor(timeDiff / (1000 * 3600 * 24));
+        
+        if (diffDays === 0) {
+            return $_("today");
+        } else if (diffDays > 0 && diffDays < 7) {
+            // use diffDays as a key to get the correct translation
+            let message = diffDays === 1 ? "day_ago" : "days_ago";
+             return `${$_(message, { values: { days: diffDays } })}`;
+        } else if (diffDays >= 7 && diffDays < 30) {
+            let message = Math.floor(diffDays / 7) === 1 ? "week_ago" : "weeks_ago";
+            return `${$_(message, { values: { weeks: Math.floor(diffDays / 7) } })}`;
+        } else if (diffDays >= 30 && diffDays < 365) {
+            let message = Math.floor(diffDays / 30) === 1 ? "month_ago" : "months_ago";
+            return `${$_(message, { values: { months: Math.floor(diffDays / 30) } })}`;
+        } else if (diffDays >= 365) {
+            let message = Math.floor(diffDays / 365) === 1 ? "year_ago" : "years_ago";
+            return `${$_(message, { values: { years: Math.floor(diffDays / 365) } })}`;
+        }
+    };
+
+    let shortBody: string = body.slice(0, short_body_char_limit).trim();
+  </script>
+  
+  <section class="review-container" data-testid="container">
+    {#if clickable}
+        <!-- svelte-ignore a11y-missing-content -->
+        <a class="review-link" data-testid="link" href={link}></a>
```

Comment:
sí, es buena idea tener propiedad aparte para testing.
No sé porque tiene link. El unico caso clickable es cuando el texto es muy largo, y el usuario apreta more button para agrandar el coponente. Pero no hay cambio en url.

Reply:
El link es por este item del ticket: 
- The review is clickable on the images page but not on the reviews page.

Entendí que el component era clickable, o la idea es hacer el contenedor de review page clickable?

Reply:
Esa linea es viejo, el unico elemento que es cliquable es "more"